assets/src/js/toggle_comment_submit.js

```diff
@@ -1,17 +1,33 @@
+/**
+ * Handles enabling/disabling the comment form submit button
+ * based on GDPR checkbox state and Cloudflare Turnstile validation.
+ */
 document.addEventListener('DOMContentLoaded', () => {
   const checkbox = document.querySelector('#gdpr-comments-checkbox');
   const submit = document.querySelector('#commentform button[type="submit"]');
 
-  if (!checkbox || !submit) {
-    return;
+  if (!checkbox || !submit) {return;}
+
+  const turnstile = document.querySelector('.cf-turnstile');
+  let turnstileValid = true;
+
+  if (turnstile) {
```

Comment:
Can't we add a check beforehand with the others whether `turnstile` is defined or not?

Reply:
Nevermind, I just understood that the function continues afterwards 🤦‍♀️